packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapHeatmapController.m

```diff
@@ -0,0 +1,158 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTGoogleMapHeatmapController.h"
+#import "FLTGoogleMapJSONConversions.h"
+@import GoogleMapsUtils;
+
+@interface FLTGoogleMapHeatmapController ()
```

Comment:
This new class needs tests.

Reply:
You found the XCTests, and there's also black-box UI XCUITest integration tests at https://github.com/flutter/plugins/blob/main/packages/google_maps_flutter/google_maps_flutter_ios/example/ios/RunnerUITests/GoogleMapsUITests.m

Reply:
Done